pkg/benchmarkjunit/OWNERS

```diff
@@ -0,0 +1,4 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- cjwagner
```

Comment:
Maybe pick at least one other victim to help you review?

Reply:
Added you :stuck_out_tongue: